include/psa/crypto_sizes.h

```diff
@@ -270,7 +270,7 @@
 #elif defined(PSA_WANT_ECC_MONTGOMERY_448)
 #define PSA_VENDOR_ECC_MAX_CURVE_BITS 448u
 #elif defined(PSA_WANT_ECC_TWISTED_EDWARDS_448)
-#define PSA_VENDOR_ECC_MAX_CURVE_BITS 448u
```

Comment:
Wait, this must be moved above the PSA_WANT_ECC_MONTGOMERY_448 since it's bigger